src/App.tsx

```diff
@@ -1095,1225 +2385,2408 @@ const App = () => {
+                          ...s,
+                          [calendarTab]: [...s[calendarTab], task],
+                        }));
+                        return;
+                      }
 
-                  <div style={{marginRight: "40px", marginBottom: "40px"}}>
-                    <demo.SmallLabel>No<br />Text</demo.SmallLabel>
-                    <DefaultStar noText />
-                    {DemoCodeBlock({code: `<DefaultStar noText />`})}
+                      setTasks((s) => ({
+                        ...s,
+                        [calendarTab]: [
+                          ...s[calendarTab].slice(0, foundIdx),
+                          task,
+                          ...s[calendarTab].slice(foundIdx + 1),
+                        ],
+                      }));
+                    }}
+                    footerTasks={footerTasks.filter((v) => !v.completed)}
+                    headerTabs={[
+                      {
+                        content: "",
+                        label: "All Tasks",
+                        onClick: () => {
+                          setCalendarTab("all");
+                          console.log("all tasks");
+                        },
+                      },
+                      {
+                        content: "",
+                        label: "Client Tasks",
+                        onClick: () => {
+                          setCalendarTab("client");
+                          console.log("client tasks");
+                        },
+                      },
+                      {
+                        content: "",
+                        label: "Project Tasks",
+                        onClick: () => {
+                          setCalendarTab("project");
+                          console.log("project tasks");
+                        },
+                      },
+                      {
+                        content: "",
+                        label: "Other Tasks",
+                        onClick: () => {
+                          setCalendarTab("other");
+                          console.log("other tasks");
+                        },
+                      },
+                    ]}
+                    views={[
+                      {
+                        type: "TASK",
+                        title: "Tasks",
+                        selected: true,
+                        Icon: icons.UsersIcon,
+                      },
+                      { type: "PROJECT", title: "Projects", selected: false },
+                      { type: "PO", title: "Pos", selected: false },
+                    ]}
+                    onClickView={() => {}}
+                  />
+                </demo.InnerContainer>
+              </demo.OuterContainer>
+
+              <demo.OuterContainer title="Pagination" id="pagination">
+                <demo.InnerContainer noBottomLine>
+                  <Pagination currentPage={1} totalPages={4} onChange={(page:number) => 0}/>
```

Comment:
Just to demonstrate the functionality, could you add the following to the top of the component somewhere:

```ts
const [page, setPage] = useState(0);
```

And then wire it up:

```ts
<Pagination currentPage={page} totalPages={4} onChange={setPage}/>
```